### ISSUE TITLE
When the version of null safety will be released ?

### ISSUE THREAD

**Issue by medmsd** [anirudhsharma392/Expansion-Card#17]

**Comment by keanallen** [comment]
Any update on this?

**Comment by prakasharyaman** [comment]
When the version of null safety will be released ??

**Comment by mikwee** [comment]
Since this package hasn't been updated in 9 months, I guess we may have to look for other solutions. Such a shame.

**Comment by guyluz11** [comment]
TLDR; The last version didn't get uploaded to pub.dev but it has null safety here in master branch.

So in the meantime guys we can use this package by adding it in `pubspec.yaml` like this 

```
  expansion_card:
    git: https://github.com/anirudhsharma392/Expansion-Card.git
```

Long Story;

Interesting, so I went on the code and found out that ...
This package does support null safety!!.

On 06/03/2021 a new commit to the `pubspec.yaml` upgraded environment SDK to 2.12.0 and above.
https://github.com/anirudhsharma392/Expansion-Card/blob/39d47fa029c19ac38185637d3c0926cb4bb25b14/pubspec.yaml#L8

And running the migration tool show that it is already got migrated 
![image](https://user-images.githubusercontent.com/9304740/148681554-4bb11e6f-5c9c-4c42-8ebe-4d5ade94e1ef.png)

But on pub.dev it shows that it is still on 2.1.0 which is not enoght
![image](https://user-images.githubusercontent.com/9304740/148681652-18603071-8625-45bc-981d-c6b55c29c889.png)


So @anirudhsharma392 can you upload the last version to pub dev?.
I understand that you are busy but it does not look like a lot of work to just upload the last version.

Thanks in advance.
